Update onto-docker.md (#7)
materialdigital/pmd-server: docs/pages/services/onto-docker.md

cd ontodocker
```

### 2. Connect to SSO Identity Provider (IDP)

> If you alreday have a valid client_secrets.json already, you must copy it to ```./data/oidc/```

Reviewer: ```suggestion
> **Note:** If you have a valid client_secrets.json already, place it in `./data/oidc/`
{: .warning }
```

Author: The triple backticks are a mistake. Single backticks are the right way, I agree.

> Otherwise in order to connect OntoDocker to the IDP you might need an initial access token (IAT) or generate one, if you want to connect it to your local Instance. (see "Initial Access Token" section of the [Keycloak manual](https://www.keycloak.org/docs/latest/securing_apps/#_initial_access_token))

Reviewer: ```suggestion
In order to connect OntoDocker to the IDP you might need an initial access token (IAT), which you will receive from the maintainer of the IDP,  or generate one, if you want to connect it to your local Instance. (see "Initial Access Token" section of the [Keycloak manual](https://www.keycloak.org/docs/latest/securing_apps/#_initial_access_token))
```

we should probably add a section about IAT's in our deployment guide but for now Keycloak manual is also fine

Save the Initial Access Token as ```ia.jwt```

Replace KEYCLOAK_URL, REALM_NAME in ```provider_info.json``` accordingly to the IDPs properties

Set the APPLICATION_URL to the url where your instance is supposed to be accessible at

> Copy both ```ia.jwt``` and the customized ```provider_info.json``` files to ```./data/oidc/```

Reviewer: Add some point we should probably try to have all configuration parameters in a single config.json and use this to automatically populate the custom config files

Author: Good idea
